Assets/Code/Player.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+	public Vector2 inputVec;
+	public float speed;
+		
+	Rigidbody2D rigid;
+		
+	void Start()
+	{
+		rigid = GetComponent<Rigidbody2D>();
+	}
```

Comment:
```suggestion
	void Start()
	{
		// Rigidbody2D 초기화
		rigid = GetComponent<Rigidbody2D>();
	}
```
Rigidbody2D 초기화에 대한 주석 추가